app.js

```diff
@@ -1,23 +1,27 @@
 var userLibrary = [];
-var tempNoteId;
-// this control flow allows us to avoid errors when switching between html pages
+var tempNoteId; // this control flow allows us to avoid errors when switching between html pages
 
-// if a user visits the site on index.html and userIndex and userLibrary both exist in local storage, parse the info and save it to the UserLibrary variable
-if (document.title === "Welcome to Note Fellows!") { //index.html
-  if (localStorage.userIndex  && localStorage.userLibrary) {
+if (document.title === 'Welcome to Note Fellows!') { // index.html
+  if (localStorage.userIndex && localStorage.userLibrary) {
     var userIndex = JSON.parse(localStorage.getItem('userIndex'));
     userLibrary = JSON.parse(localStorage.getItem('userLibrary'));
+    $('#loginForm').append(returnUserForm('loginTemplate', 'returningUserContext'));
```

Comment:
right now returnUserForm doesn't take two parameters so not sure what 'loginTemplate' and 'returningUserContext' are doing here.